paimon-format/src/test/java/org/apache/paimon/format/parquet/ParquetReadWriteTest.java

```diff
@@ -834,11 +840,14 @@ private Path createNestedDataByOriginWriter(int rowNum, File tmpDir, int rowGrou
         MessageType schema =
                 ParquetSchemaConverter.convertToParquetMessageType(
                         "paimon-parquet", NESTED_ARRAY_MAP_TYPE);
+        String[] candidates = new String[] {"snappy", "zstd", "gzip"};
+        String compress = candidates[new Random().nextInt(3)];
```

Comment:
I know this was intended to balance coverage and test running time, but using random in test cases is usually not ideal. Let's use a parameterized test here for these three formats.

Reply:
Sorry to reply so late, this is refer to the previous pr, i will change it to the parameterized test